js/ui/appDisplay.js

```diff
@@ -218,6 +221,24 @@ const BaseAppView = new Lang.Class({
         return [movedList, removedList];
     },
 
+    _findAddedIcons: function() {
+        let oldItemLayout = this._allItems.map(function(icon) { return icon.getId(); });
+        if (oldItemLayout.length === 0) return [];
```

Comment:
Coding style: put the return on a new line.

Reply:
Oops